react-native/firebase/debug-events.md

```diff
@@ -0,0 +1,156 @@
+# [Standard] Good practices to track events in Firebase
+
+## Owner: [Thomas Pucci](https://github.com/tpucci)
+
+## Prerequisites
+
+* [React Native Firebase](https://github.com/invertase/react-native-firebase) is installed on your project
+* You want to track one event
+
+## Scrum
+
+* You should have one ticket to send the event to Firebase which your PO will validate with a screenshot.
```

Comment:
Maybe "Validating events tracking with Firebase should be done with two tickets:
1. ...
2. ...
"

Reply:
👍